src/Uno.UI/UI/Xaml/DependencyPropertyDetailsCollection.cs

```diff
@@ -214,9 +232,25 @@ private void ReturnEntriesToPool()
 
 		internal IEnumerable<DependencyPropertyDetails> GetAllDetails() => Entries.Trim();
 
+		/// <summary>
+		/// Adds a default value provider.
+		/// </summary>
+		/// <param name="provider">Default value provider.</param>
+		/// <remarks>
+		/// Providers which are registered later have higher precedence.
```

Comment:
The term _precedence_ is confusing here because it's already used in the context of dependency properties and it has a very specific meaning.
```suggestion
		/// Providers which are registered later have higher priority.
```